api/.dependency-cruiser.js

```diff
@@ -75,11 +83,11 @@ module.exports = {
       to: { path: "winston-cloudwatch" },
     },
     {
-      from: { path: "src/db" },
+      from: { path: "src/db", path: "src/functions/messagingService" },
       to: { path: "fs" },
     },
     {
-      from: { path: "src/db" },
+      from: { path: "src/db", path: "src/functions/messagingService" },
```

Comment:
The current dependency-cruiser rules aren’t doing what we intend.
They define path twice inside the same from object, which means only the last one is actually used:
```suggestion
     {
  from: { path: ["src/db", "src/functions/messagingService"] },
  to: { path: "fs" },
},
{
  from: { path: ["src/db", "src/functions/messagingService"] },
  to: { path: "path" },
},
```

Reply:
Ah, thank you for catching that! I realized that these also no longer need to be modified because we are no longer using these libraries in messagingService/app.ts, so I'm gonna revert this!